docs/modules/ROOT/pages/ios_collaboration.adoc

```diff
@@ -0,0 +1,83 @@
+= Collaboration and Links
+:toc: right
+
+This section shows how to collaborate with other users and how to work with links.
+
+== Collaborate With Other Users on Files and Folders
+
+image:31_Collab.png[Collaborate With Other Users on Folders and Files]
+
+The Mobile App for iOS supports the following collaboration functionality:
+
+* Adding users and grant permissions
+* Adding groups and grant permissions
+* Changing permissions permissions
+* Deleting collaborators
+
+== Grant Access With Links
+
+Just as with the web interface, the iOS app lets you grant access to both files and folders with links.
+Specifically, you can do the following:
+
+* xref:copy-public-links[Copy public links]
+* xref:create-private-links[Create private links]
+* xref:set-passwords-and-expiration-dates[Set passwords and expiration dates]
+* xref:send-private-link-urls-to-other-apps[Send private link URLs to other apps (mail, messenger)]
+* xref:delete-private-links[Delete private links]
+
+=== Copy Public Links
```

Comment:
As suggestion, i'd set the `Copy Link` section below the `Creation section` . First you create, then you can copy

Reply:
Should be resolved with 4ea032d546836cef88ce32fe06ed3484b4f5d068